src/main/java/io/vertx/ext/dropwizard/impl/VertxMetricsImpl.java

```diff
@@ -52,6 +48,8 @@
  * @author <a href="mailto:nscavell@redhat.com">Nick Scavelli</a>
  */
 class VertxMetricsImpl extends AbstractMetrics implements VertxMetrics {
+  private static final ThreadMXBean threadMXBean = ManagementFactory.getThreadMXBean();
+  private static int eventLoopThreads;
```

Comment:
it should not be static otherwise it will report the total number of event loop thread per JVM.